src/specklepy/core/api/models/__init__.py

```diff
@@ -68,4 +70,8 @@
     "Streams",
     "Activity",
     "ActivityCollection",
+    "InstanceProxy",
```

Comment:
I see some proxy classes were added, but looks like `renderMaterialProxy` is missing

Reply:
Currently it relies on `RenderMaterial` which is in `Objects`, not `Core`. Both would have to be moved to `Core` if so

Reply:
yep, noticed it, added it to the same place as in C#